sensor_msgs/Image[] image_array